docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -56,36 +50,33 @@ To get started, refer to
 If you have a log file or delimited CSV, TSV, or JSON file, you can upload it,
 view its fields and metrics, and optionally import it into {es}.
 
-NOTE: This feature is not intended for use as part of a repeated production
+NOTE: The upload feature is not intended for use as part of a repeated production
 process, but rather for the initial exploration of your data.
 
-You can upload a file up to 100 MB. This value is configurable up to 1 GB in
-<<fileupload-maxfilesize,Advanced Settings>>. To upload a file with geospatial
-data, refer to <<import-geospatial-data,Import geospatial data>>.
-
-[role="screenshot"]
-image::images/add-data-fv.png[Uploading a file in {kib}]
-
-The {stack-security-features} provide roles and privileges that control which
-users can upload files. To upload a file in {kib} and import it into an {es}
-index, you'll need:
-
+. To upload a file, you’ll need the following privileges,
+which you can manage in **{stack-manage-app}**.
++
```

Comment:
I think there's another requirement now per https://github.com/elastic/stack-docs/pull/1858#discussion_r741841390
```suggestion
* [ ] `Data Views Management: All` {kib} feature privileges
```
The cluster privilege detail also differs a bit in https://www.elastic.co/guide/en/machine-learning/master/setup.html